src/apply.ts

```diff
@@ -269,56 +179,14 @@ export async function apply(options: ReadOptions): Promise<number> {
 			},
 		});
 
-		// Requesting for a token fails if API_TOKEN is not provided.
-		// let additionalArguments: string[] = [];
-		// if (await balenaCloudSdk.auth.isLoggedIn()) {
-		// 	const remoteAuthResponse = await balenaCloudSdk.request.send({
-		// 		baseUrl: await balenaCloudSdk.settings.get('apiUrl'),
-		// 		url: '/auth/v1/token',
-		// 		qs: {
-		// 			service: remoteImage.registry,
-		// 			scope: `repository:${remoteImage.repository}:pull`,
-		// 		},
-		// 	});
-		// 	additionalArguments = [
-		// 		'--src-registry-token',
-		// 		remoteAuthResponse.body.token,
-		// 	];
-		// }
-
-		// There is currently a bug in the SDK that does not return the correct API URL
-		// so we can't use `await sdk.settings.get('apiUrl');`
-		// const localRegistryUrl = await sdk.settings.get('registry2Url');
-		// const localAuthResponse = await sdk.request.send({
-		// 	baseUrl: await sdk.settings.get('apiUrl'),
-		// 	url: '/auth/v1/token',
-		// 	qs: {
-		// 		service: localRegistryUrl,
-		// 		scope: `repository:v2/${
-		// 			localImage.is_stored_at__image_location.split('/v2/')[1]
-		// 		}:pull,push`,
-		// 	},
-		// });
-		// const result = await runCommand('skopeo', [
-		// 	'copy',
-		// 	'--override-os',
-		// 	'linux',
-		// 	...additionalArguments,
-		// 	'--dest-registry-token',
-		// 	localAuthResponse.body.token,
-		// 	`docker://${remoteImage.image_name}@${remoteImage.content_hash}`,
-		// 	`docker://${localRegistryUrl}/v2/${
-		// 		localImage.is_stored_at__image_location.split('/v2/')[1]
-		// 	}:latest`,
-		// ]);
-		// console.log(result);
+		// TODO: upload images from the release bundle at this part
 
-		// mark image as successful
+		// // mark image as successful
 		await sdk.pine.patch({
 			resource: 'image',
 			id: localImage.id,
 			body: {
-				status: remoteImage.image[0].status,
+				status: 'success',
```

Comment:
Again, you’re making an assumption about a value that you haven’t proven is safe to make. What prevents the actual status of the image to be anything other than “success”?